fanout/fanout-service.js

```diff
@@ -21,6 +16,12 @@ require('seneca')({
   })
 
   .ready(function(){
+    this.add('info:entry', function(msg,done){
```

Comment:
Unfortunately this is needed by the transport-configuration-plugin to avoid circular references.

I've built that plugin to create clients for every other service, not only the ones really used (this basically replicate the same behaviour of seneca mesh).

The problem though is that this service will both have a client with `pin:'info:entry'` and and action for `info:entry`. If the client is added last its info:entry will overwrite the local `info:entry` resulting in the service receiving the message from the transport and then resending it to index-service. index-service itself will have the same problem and will re-send again the message resulting in an infinite loop.

If the local action is added after the plugin it will prevail and when seneca will receive info:entry from the transport will follow the intended path.

I tried alternative approaches, but couldn't find one that broke the infinite loop (without major changes to ramanujan or the transport-configuration-plugin)